apartments/tests.py

```diff
@@ -61,6 +61,11 @@ def test_get_apartment(self, new_apartment):
         check_apartment = Apartment.objects.get(owner=new_apartment.owner)
         assert check_apartment == new_apartment
 
+    def test_get_all_relevant_apartments(self):
+        apartments_qurey = Apartment.get_all_relevant_apartments()
+        assert all(isinstance(current_apartment, Apartment) for current_apartment in apartments_qurey)
+        assert all(current_apartment.is_relevant is True for current_apartment in apartments_qurey)
```

Comment:
Pretty sure `is True` can be discarded here (Truthy value).

Reply:
Got it